Tests/ReadyCampaignDispatcherSpec.swift

```diff
@@ -179,6 +190,26 @@ class ReadyCampaignDispatcherSpec: QuickSpec {
                             }
                         }
                     }
+
+                    context("httpSession errors on loading imageUrl") {
+                        beforeEach {
+                            // swiftlint:disable:next force_cast
+                            let httpSession = dispatcher.httpSession as! URLSessionMock
+                            httpSession.httpResponse = HTTPURLResponse(url: URL(string: "https://example.com/cat.jpg")!,
+                                                                       statusCode: 500,
+                                                                       httpVersion: nil,
+                                                                       headerFields: nil)
+                            httpSession.responseError = NSError(domain: "campaign.error.test", code: 1, userInfo: nil)
+                        }
+
+                        it("won't display campaign when imageUrl is defined") {
```

Comment:
Please add also one test for positive case (i.e. image defined and status code 200)